modules/Core/hooks/ContentHook.php

```diff
@@ -10,7 +10,7 @@
 class ContentHook extends HookBase {
 
     public static function codeTransform(array $params = []): array {
-        if (parent::validateParams($params, ['content'])) {
+        if (Config::get('core.encoded_content_compat') && parent::validateParams($params, ['content'])) {
```

Comment:
Should not be disabled here, As its a function any module can use
Disable the listener for the forum content instead

Reply:
I'd argue that codeTransform should be disabled for all modules using it

Reply:
Yet these 2 functions are functions modules may wanna use for other things

But yea at the same time it suck that modules must self disable it then but no other way to do it i guess other than making 2 new codeTransform and decode functions

Reply:
What you think @samerton

Reply:
This function actually needs to be removed as per the following Discord conversation: https://discord.com/channels/246705793066467328/246705793066467328/1071149195165175950